include/trace/events/sof.h

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause)
+//
+// This file is provided under a dual BSD/GPLv2 license.  When using or
+// redistributing this file, you may do so under either license.
```

Comment:
Humm, I think this is completely dependent on the Linux trace subsystem, so we should probably choose GPL-2.0-only for the license. I don't think it makes sense to have a BSD-3-Clause here.

@ujfalusi @ranj063 @lgirdwood comments?

Reply:
@plbossart this is what I told @nklayman as well. It should be GPL2.0-only too I think